cmd/system-probe/api/config.go

```diff
@@ -7,11 +7,9 @@ import (
 )
 
 // setupConfigHandlers adds the specific handlers for /config endpoints
-func setupConfigHandlers(r *mux.Router) *mux.Router {
-	r.HandleFunc("/", settingshttp.Server.GetFull(config.Namespace)).Methods("GET")
-	r.HandleFunc("/list-runtime", settingshttp.Server.ListConfigurable).Methods("GET")
-	r.HandleFunc("/{setting}", settingshttp.Server.GetValue).Methods("GET")
-	r.HandleFunc("/{setting}", settingshttp.Server.SetValue).Methods("POST")
-
-	return r
+func setupConfigHandlers(r *mux.Router) {
+	r.HandleFunc("/config/", settingshttp.Server.GetFull(config.Namespace)).Methods("GET")
```

Comment:
I checked out this branch & tested it, & running `system-probe config` gives me a 404; this should fix it though.
```suggestion
	r.HandleFunc("/config", settingshttp.Server.GetFull(config.Namespace)).Methods("GET")
```

Reply:
Thanks for catching this!